drbitboy/SpiceyPy
# Reduce test kernel set by over 400MB, down to < 20MB.

spiceypy/tests/gettestkernels.py

* Replace de421.bsp with smaller de405s.bsp (DE-405 short ca. 1997 to 2010).
* DE405S:  Needs to be <native>-IEEE, typically LTL-IEEE for Intel.
* DE405S:  Wrote script make_de405s_native.py to convert from BIG-IEEE kernel at JPL/NAIF.
* DE405S:  Integrate make_de405s_native.py script into gettestkernels.py.
* DE405S:  Shift many time-lookup calls into range of de405s.bsp.
* DE405S:  Add step to spiceypy/tests/gettestkernels.py to convert BIG-IEEE de405s.bsp from NAIF to native byteorder.
* Shift Cassini kernel coverage from 2004 to 2013
* CASSINI/2013:  Change ABCORR to LT from LT+S, to NONE from S
* Drop MER and MRO kernels; use Phobos instead of MRO/MER; hardcode MER location
* Use smaller Phobos DSK
* Use Mercury transit of Sun instead ofr MRO transit of Mars
* Use smaller Earth rotation state binary PCK
* Remove latest Earth binary PCK; it changes too often
* Remove MarsKernels class
* Remove obsolete cleanup and download (get*Kernels) routines and calls
* Remove remove redundant kernelFile.close()
* Simplify global downloadTestKernels routine

spiceypy/tests/make_de405s_native.py

* New script to convert de405s.bsp from NAIF to native byte-order
* DE405S:  Does functionally the same thing as SPICE utility BINGO
* DE405S:  Downloads NAIF de405s.bsp as de405s_bigendian.bsp
* DE405S:  Converts de405s_bigendian.bsp to def05s.bsp
* DE405S:  Asserts with checksum to ensure success

spiceypy/tests/test_wrapper.py

* Adjust assertion expected values to match returns from new kernels
* Increase linewidth of numpy default print options
* Adjust parameters in test_dskw02_dskrb2_dskmi2()
* Remove obsolete cleanup calls
* Replace MGS/Mars tests with Cassini/Enceladus tests; adjust expected values
* test_ckcov():  simplify assertion, improve test
* Change ['%s%f%d' % tuple(...)] format statements to ['{}{}{}'.format(...)
